app/code/Magento/InventoryApi/Test/Api/StockRepository/ValidationTest.php

```diff
@@ -125,6 +128,63 @@ public function testFailedValidationOnUpdate(string $field, $value, array $expec
         $this->webApiCall($serviceInfo, $data, $expectedErrorData);
     }
 
+    /**
+     * @param string|null $value
+     * @param array       $expectedErrorData
+     *
+     * @dataProvider          failedNonExistedWebsiteValidationDataProvider
+     * @magentoApiDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/stock.php
+     */
+    public function testNonExistedWebsiteValidationOnUpdate($value, array $expectedErrorData)
+    {
+        $stockId = 10;
+        $stockName = 'stock-name-1';
+        $data = [
+            "stock_id" => $stockId,
+            "name" => $stockName,
+            "extension_attributes" => [
+                "sales_channels" => [
+                    ["type" => "website", "code" => $value]
```

Comment:
Pls, 
1) use constant instead of website
2) use 2 website codes (real and not existed) You can put `sales_channels` in data provider (or remove dataprovider because of we have only one case now)

Reply:
done.
1. constant used
2. added dataprovider with 2 sales channel to the test